### PR TITLE
Begin addding `flake8-simplify` checks

### PR DESCRIPTION
**Pull request by danieleades**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length = 95
-ignore = E116,E241,E251,E741,W504,I101,B006,B023
+ignore = E116,E241,E251,E741,W504,I101,B006,B023,SIM102,SIM103,SIM105,SIM110,SIM113,SIM114,SIM115,SIM117,SIM223,SIM300,SIM401,SIM904,SIM905,SIM907
```
Comment (danieleades) [inline setup.cfg:3]: this is silencing every flake8-simplify warning *except* SIM104

**Comment by danieleades** [comment]
i had some previous flake8-simplify PRs, but i thought it might be better to split these up and discuss separately. Some of them were a little controversial.

**Comment by danieleades** [comment]
i'll trade you these PRs for some knowledge ;)

https://stackoverflow.com/q/73700692/17539451

**Comment by AA-Turner** [comment]
The stackoverflow question doesn't exist? What can I help with?

A

**Comment by danieleades** [comment]
> The stackoverflow question doesn't exist? What can I help with?
> 
> A

Ah thanks anyway, I managed to resolve my issue

**Comment by AA-Turner** [comment]
Thanks Dan!

A

**Pull request merged**